### PR TITLE
screen-saver-control: Add version 1.61

### PR DESCRIPTION
**Pull request by wenmin92**

### CODE REVIEW

**Review by Ash258 — CHANGES_REQUESTED** [review]
No manifest in official buckets uses snake case naming convention.

File: bucket/screen-saver-control.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "shortcuts": [
+        [
+            "ScreenSaverControl.exe",
+            "Screen Saver Control"
+        ]
+    ],
+    "checkver": "Version ([\\d.]+)",
+    "autoupdate": {
+        "url": "http://www.ampsoft.net/files/ScreenSaverControl.zip"
+    }
```
Comment (Ash258) [inline bucket/screen-saver-control.json:18]: Latest version 2009

```suggestion
    ]
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/screen-saver-control.json (hunk 1 above)
Comment (wenmin92) [reply bucket/screen-saver-control.json:18]: Thanks!

Comment (Ash258) [reply bucket/screen-saver-control.json:18]: Stop resolving conversations, when you do not commit changes...

Comment (Ash258) [reply bucket/screen-saver-control.json]: Commenting on resolved conversations is nonsense as it does not get revealed, so it is completely waste notification, which noone would see...

Comment (wenmin92) [reply bucket/screen-saver-control.json]: Thank you for your guidance :blush:

File: bucket/screen-saver-control.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    "version": "1.61",
+    "description": "A small utility that allows deactivate or activate both screen saver and monitor power off with a simple click.",
+    "homepage": "http://www.ampsoft.net/utilities/ScreenSaverControl.php",
+    "license": "Freeware",
+    "url": "http://www.ampsoft.net/files/ScreenSaverControl.zip",
+    "hash": "26ab07326d91db6a80fc9de77dedbc9b36bb3070c514a1be7239ea397beb512f",
+    "bin": "ScreenSaverControl.exe",
```
Comment (Ash258) [inline bucket/screen-saver-control.json:8]: What are available paramaters?

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/screen-saver-control.json (hunk 2 above)
Comment (wenmin92) [reply bucket/screen-saver-control.json:8]: image

**Review by Ash258 — APPROVED** [review]

**Review by rasa — APPROVED** [review]

**Pull request merged**